components/elm/src/external_models/emi/src/elm_stub/elm_types_stub/clm_instMod.F90

```diff
@@ -1,4 +1,4 @@
-module clm_instMod
```

Comment:
Should the filename be renamed?

Reply:
[Same as above](https://github.com/E3SM-Project/E3SM/pull/3842#discussion_r535744540)